### ISSUE TITLE
Orca Version with 3 digits

### ISSUE THREAD

**Issue by Cybert92** [grimme-lab/enso#1]
Apparently,  `enso.py`  does not accept an Orca Version with 3 digits. 

**Problem:**
In` .ensorc`: `ORCA version: 4.2.1`

gives:

```
Traceback (most recent call last):
  File "/home/seibert/bin/enso.py", line 6053, in read_program_paths
    if float(line.split()[2]) < 4.1:
ValueError: could not convert string to float: '4.2.1'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/seibert/bin/enso.py", line 11126, in <module>
    ) = enso_startup(cwd)
  File "/home/seibert/bin/enso.py", line 7722, in enso_startup
    ) = ensorcsettings.read_program_paths(ensorc)
  File "/home/seibert/bin/enso.py", line 6060, in read_program_paths
    if args.prog == "orca" or args.prog4 == "orca":
NameError: name 'args' is not defined
```

**Comment by fabothch** [comment]
Hi, I will look into this! The ORCA-Version information is anyway only necessary, because ORCA changed some keywords for the solvent.

**Comment by fabothch** [comment]
Fixed in ENSO v1.27 see #2
